repentogon/Patches/ModsMenuStuff.cpp

```diff
@@ -46,7 +46,7 @@ unordered_map<int, int> lastKeyPressTimeMap;
 bool cursorblink = false;
 float prevscroll = 0;
 int prevselect = 0;
-int minmods = 15;
+const unsigned int minmods = 15;
```

Comment:
Why const exactly ? This feels like a workaround for a warning rather than a semantic choice

Reply:
reverted const, organized variable definition block a bit